HW11/src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,60 @@
+package coffee.order;
+
+import java.util.ArrayList;
+
+
+public class CoffeeOrderBoard {
+    private ArrayList<Order> orders;
+
+    public CoffeeOrderBoard() {
+        orders = new ArrayList<>();
+    }
+
+    public void add(String name) {
+        int orderNumber = orders.size() + 1;
```

Comment:
Let's imagine the next situation:
1. Coffee shop gets 3 orders - `orderNumer` of those will be 2, 3 and 4 because `orders.size()+1` 
2. Coffe shop delivered 2 orders, which means that `orders.size()` will be equal to 1
3. Then Coffe Shop gets 1 more order, and here, according to logic - the `orderNumber` of this order will be 2 - because `orders.size()+1`

This logic breaks a 3rd requirement of the task.